src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AbstractRazorDelegatingEndpoint.cs

```diff
@@ -105,7 +117,28 @@ protected virtual Task<TResponse> HandleDelegatedResponseAsync(TResponse delegat
             return default;
         }
 
-        var projection = await _documentMappingService.TryGetProjectionAsync(documentContext, request.Position, requestContext.Logger, cancellationToken).ConfigureAwait(false);
+        var position = request.Position;
```

Comment:
maybe instead of this we just have a `protected virtual Position ModifyRequestProjectedPosition(...)`?

Reply:
The only reason to avoid this is that there are a few handlers that would need the logic. I would do it this way if I could come up with a reasonable name for a class those handlers could inherit from, that sat in the middle of the hierarchy